wp-content/themes/core/components/accordion/js/accordion.js

```diff
@@ -79,6 +79,11 @@ const openAccordion = ( header, content ) => {
 	setOffset();
 
 	slide.down( content, content.id, options.speed );
+
+	if ( ! el.container[ 0 ].dataset.scrollto ) {
```

Comment:
Unfortunately, this selector won't work in instances where more than one accordion component is used on the page. `el.container[ 0 ]` will always be the first, regardless of which one is clicked.

You'll probably want to set a new const for the _current_ component using `closest()` and then check that for the data attr.